docs(community): reposition community and support docs to reflect both products (#77661)
airbytehq/airbyte: docs/community/getting-support.md

# Get support

> Airbyte offers support for both its data replication platform and Airbyte Agents. If you can't find an answer to your question in the docs, use one of the following options.

Author: ```suggestion
Airbyte offers support for both Data Replication and Airbyte Agents. If you can't find an answer to your question in the docs, use one of the following options.
```

## Support center

**For support with any Airbyte product, create a ticket in Airbyte's [support center](https://support.airbyte.com/hc/en-us/requests/new).** Support agents are available 7 AM to 7 PM Eastern time. Response times may vary according to your plan and SLAs.

- For account or billing inquiries, [contact sales](https://airbyte.com/talk-to-sales).

> - Sign up for Airbyte with your company email address. Airbyte doesn't support personal accounts.

Author: This is only true for data replication, not agents.

If you need personalized help with Core, consider upgrading to one of Airbyte's [paid plans](https://www.airbyte.com/pricing).

### Connector support levels

Data replication connector support depends on the connector's certification status. See [Connector Support Levels](/integrations/connector-support-levels) for details.

If you don't see a data replication connector you need, submit a [connector request](https://airbyte.com/connector-requests).

## Airbyte Agents

Airbyte Agents support depends on your plan. Free plan users have access to AI and community support. Individual and Team plan users also receive standard human support. Custom plan users receive dedicated human support.

For full details on Agents plans and support availability, see [Billing and pricing](/ai-agents/admin/billing).

## Community resources

These resources are available to all Airbyte users regardless of product or plan.

### Community Slack

> [Join the Slack community](https://slack.airbyte.com) for help from other users and Airbyte team members. Popular channels are `#ask-ai` and `#ask-community-for-troubleshooting`.

Author: #airbyte-agents, #airbyte-agents-ask-ai

### GitHub discussions

[GitHub discussions](https://github.com/airbytehq/airbyte/discussions) are a great way to propose, vote on, and discuss Airbyte improvements.

### Report an issue

If you've found a bug or issue with Airbyte or its documentation, [create an issue](https://github.com/airbytehq/airbyte/issues/new/choose) on GitHub. If you're willing to work on this issue yourself, please indicate this.

Check the [issue list](https://github.com/airbytehq/airbyte/issues) first in case the issue has already been reported.

### Public roadmap

> To stay updated on Airbyte's future plans, take a look at [the roadmap](https://github.com/orgs/airbytehq/projects/37/views/1). Airbyte updates this every few weeks with initiatives that are of interest to the community.

Author: Let's not discuss the public roadmap. We haven't been updating it.

# Get support

> Airbyte offers support for both Data Replication and Airbyte Agents. If you can't find an answer to your question in the docs, use one of the following options.

Reviewer: ```suggestion
Airbyte offers support for both Data Replication and Airbyte Agents. If you can't find an answer to your question in the docs, use one of the following options:
```

## Data Replication

> ### In-app support bot

Reviewer: ```suggestion
### In-app support agent bot
```

> **The in-app support bot is available 24/7 for Airbyte Cloud and Enterprise customers.** Look for the chat button in the bottom-right corner of the screen. The bot can help answer questions about your workspace, connections, and common troubleshooting scenarios. It has context about your current workspace, making it easier to get relevant assistance immediately. The bot can also create support tickets for the Airbyte support team if further help is needed.

Reviewer: ```suggestion
**The in-app support agent bot is available 24/7 for Airbyte Cloud and Enterprise customers.** Look for the chat button in the bottom-right corner of the screen. The bot can help answer questions about your workspace, connections, and common troubleshooting scenarios. It has context about your current workspace, making it easier to get relevant assistance immediately. The bot can also create support tickets for the Airbyte support team if further help is needed.
```

### Support center

**For additional help, create a ticket in Airbyte's [support center](https://support.airbyte.com/hc/en-us/requests/new).** Support agents are available 7 AM to 7 PM Eastern time. Response times may vary according to your plan and SLAs.

- For account or billing inquiries, [contact sales](https://airbyte.com/talk-to-sales).

> - Sign up for Airbyte Cloud with your company email address. Airbyte Cloud doesn't support personal accounts.

Reviewer: I dont think we actively block these anymore but still good to mention

Author: "We suggest signing up for Airbyte with your company email address, and not a personal one."

### Core (open source)

Core users don't have access to paid support but can use the [Community Slack](#community-slack). If you need personalized help, consider upgrading to one of Airbyte's [paid plans](https://www.airbyte.com/pricing).

### Connector support levels

> Connector support depends on the connector's certification status. See [Connector Support Levels](/integrations/connector-support-levels) for details.

Reviewer: ```suggestion
Connector support depends on the connector's support level. See [Connector Support Levels](/integrations/connector-support-levels) for details.
```

If you don't see a connector you need, submit a [connector request](https://airbyte.com/connector-requests).

### GitHub discussions

[GitHub discussions](https://github.com/airbytehq/airbyte/discussions) are a great way to propose, vote on, and discuss data replication improvements.

### Report an issue

If you've found a bug or issue with Airbyte or its documentation, [create an issue](https://github.com/airbytehq/airbyte/issues/new/choose) on GitHub.

Check the [issue list](https://github.com/airbytehq/airbyte/issues) first in case the issue has already been reported.

## Airbyte Agents

### In-app support bot

> **The in-app support bot is available 24/7 for all Airbyte Agents users.** Look for the chat button in the bottom-right corner of the screen. The bot can help answer questions about your organization, troubleshoot common scenarios, and create support tickets if further help is needed.

Reviewer: ```suggestion
**The in-app support agentbot is available 24/7 for all Airbyte Agents users.** Look for the chat button in the bottom-right corner of the screen. The bot can help answer questions about your organization, troubleshoot common scenarios, and create support tickets if further help is needed.
```

> **The in-app support bot is available 24/7 for all Airbyte Agents users.** Look for the chat button in the bottom-right corner of the screen. The bot can help answer questions about your workspace, troubleshoot common scenarios, and create support tickets if further help is needed.

Author: ```suggestion
**The in-app support bot is available 24/7 for all Airbyte Agents users.** Look for the chat button in the bottom-right corner of the screen. The bot can help answer questions about your organization, troubleshoot common scenarios, and create support tickets if further help is needed.
```

### Support by plan

> - **Free**: AI and community support.

Author: Should link to the Community Slack section below.

> - **Free**: AI and [community](#community-slack) support.

Reviewer: we will let them use support bot for now. in the future we will let them use it but not escalate zendesk tickets from there

- **Individual / Team**: Standard human support, in addition to AI and community support. The in-app support bot can create a ticket for you, or you can create your own in the [Support Center](https://support.airbyte.com/hc/en-us/requests/new).
- **Custom**: Dedicated human support.

> For full details on Agents plans, see [Billing and pricing](/ai-agents/admin/billing).

Reviewer: ```suggestion
For full details on Airbyte Agents plans, see [Billing and pricing](/ai-agents/admin/billing).
```

# Get support

Airbyte offers support for both Data Replication and Airbyte Agents. If you can't find an answer to your question in the docs, use one of the following options.

## Data Replication

### In-app support bot

**The in-app support bot is available 24/7 for Airbyte Cloud and Enterprise customers.** Look for the chat button in the bottom-right corner of the screen. The bot can help answer questions about your workspace, connections, and common troubleshooting scenarios. It has context about your current workspace, making it easier to get relevant assistance immediately. The bot can also create support tickets for the Airbyte support team if further help is needed.

### Support center

**For additional help, create a ticket in Airbyte's [support center](https://support.airbyte.com/hc/en-us/requests/new).** Support agents are available 7 AM to 7 PM Eastern time. Response times may vary according to your plan and SLAs.

- For account or billing inquiries, [contact sales](https://airbyte.com/talk-to-sales).

> - Sign up for Airbyte with your company email address. Airbyte doesn't support personal accounts.

Author: ```suggestion
- Sign up for Airbyte Cloud with your company email address. Airbyte Cloud doesn't support personal accounts.
```

### Core (open source)

Core users don't have access to paid support but can use [community resources](#community-resources). If you need personalized help, consider upgrading to one of Airbyte's [paid plans](https://www.airbyte.com/pricing).

### Connector support levels

Connector support depends on the connector's certification status. See [Connector Support Levels](/integrations/connector-support-levels) for details.

If you don't see a connector you need, submit a [connector request](https://airbyte.com/connector-requests).

## Airbyte Agents

### In-app support bot

**The in-app support bot is available 24/7 for all Airbyte Agents users.** Look for the chat button in the bottom-right corner of the screen. The bot can help answer questions about your workspace, troubleshoot common scenarios, and create support tickets if further help is needed.

### Support by plan

- **Free**: AI and community support.

> - **Individual / Team**: Standard human support, in addition to AI and community support. Create a ticket in the [support center](https://support.airbyte.com/hc/en-us/requests/new).

Author: ```suggestion
- **Individual / Team**: Standard human support, in addition to AI and community support. The in-app support bot can create a ticket for you, or you can create your own in the [support center](https://support.airbyte.com/hc/en-us/requests/new).
```

- **Custom**: Dedicated human support.

For full details on Agents plans, see [Billing and pricing](/ai-agents/admin/billing).

## Community resources

These resources are available to all Airbyte users regardless of product or plan.

### Community Slack

[Join the Slack community](https://slack.airbyte.com) for help from other users and Airbyte team members. Popular channels include:

- Data Replication: `#ask-ai`, `#ask-community-for-troubleshooting`
- Airbyte Agents: `#airbyte-agents`, `#airbyte-agents-ask-ai`

### GitHub discussions

> [GitHub discussions](https://github.com/airbytehq/airbyte/discussions) are a great way to propose, vote on, and discuss Airbyte improvements.

Author: This is only true for data replication.

### Report an issue

> If you've found a bug or issue with Airbyte or its documentation, [create an issue](https://github.com/airbytehq/airbyte/issues/new/choose) on GitHub. If you're willing to work on this issue yourself, please indicate this.

Author: ```suggestion
If you've found a bug or issue with Airbyte or its documentation, [create an issue](https://github.com/airbytehq/airbyte/issues/new/choose) on GitHub.
```

Author: Again, this is only relevant to data replication.

> Check the [issue list](https://github.com/airbytehq/airbyte/issues) first in case the issue has already been reported.

Author: Also only relevant to data replication.
